.github/ISSUE_TEMPLATE/QUESTION.yml

```diff
@@ -0,0 +1,34 @@
+name: Bug Report
+description: File a bug report
+title: "[Question]: "
+labels: ["question", "triage"]
+
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to ask a question. We do our best to answer in a timely fashion.
+  - type: input
+    id: contact
+    attributes:
+      label: Contact Details
+      description: How can we get in touch with you if we need more info?
+      placeholder: ex. email@example.com
+    validations:
+      required: false
```

Comment:
I'd just like to start the conversation, and I don't have the strongest opinion one way or another. Asking for contact info seems a little out of place, as GitHub will already email the person if you respond to their issue, and GitHub should be a fine contact platform. Communications relevant to the issue should probably be public anyway.

Reply:
noted. I agree. I'll remove that.